3rdParty/CMakeLists.txt

```diff
@@ -214,8 +211,6 @@ foreach(ELEMENT ${IRESEARCH_PREBUILT})
 endforeach()
 
 add_subdirectory("${IRESEARCH_ROOT}" "${IRESEARCH_BUILD_DIR}")
-unset(BUILD_SHARED_LIBS) # otherwise ZLib (below) does not build on win32
-unset(BUILD_SHARED_LIBS CACHE) # otherwise ZLib (below) does not build on win32
```

Comment:
looks strange

Reply:
Can't tell if its needed. But let's see what the build results on Windows will look like.

Reply:
It looks broken anyway, because set in such case we don't enable this flag after this.

But yes, I agree we should wait windows build

Reply:
Seems to have compiled fine on Windows (although there is an unrelated test error on Windows).